include/z64save.h

```diff
@@ -522,7 +522,7 @@ typedef enum LinkAge {
 #define EVENTCHKINF_4A 0x4A
 #define EVENTCHKINF_OPENED_DOOR_OF_TIME 0x4B
 #define EVENTCHKINF_4C 0x4C
-#define EVENTCHKINF_4D 0x4D
+#define EVENTCHKINF_RAINBOW_BRIDGE_BUILT 0x4D
```

Comment:
My preference would be `EVENTCHKINF_SPAWNED_RAINBOW_BRIDGE`
Two reasons:
- This isnt a hard rule we have to follow all the time, but I think I generally prefer the form <DESCRIPTOR> <SUBJECT> for these flags. Similar to how you have `EVENTCHKINF_RESTORED_LAKE_HYLIA` in the other PR.
- I prefer "spawned" over "built" because the bridge is created out of thin air. Its not like they built it with hammer and nails (😂 ). Though, I guess "built" may be a good word for the gerudo valley bridge where it is implied that the carpenters built it by hand LOL

Reply:
I'm less a fan of saying the bridge "spawned" because it makes me think you're tracking an Actor_Spawned spawn

Reply:
It crossed my mind. I think I agree with "spawned" because other things can be spawned besides actors. Other alternatives would be `RAINBOW_BRIDGE_APPEARED` or `RAINBOW_BRIDGE_CREATED`, but spawned makes sense if we put the descriptor first. Or maybe `CREATED_RAINBOW_BRIDGE`.